dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1167,6 +1167,9 @@ def test_save_and_load(self):
             data = dp.Data(os.path.join(data_folder, test_file))
             save_profile = UnstructuredProfiler(data)
 
+            # address case where empty_line_count is 0
+            save_profile._empty_line_count = 1
```

Comment:
Is this comment incorrect? It says 0 but you set it to 1

Reply:
updated comment